src/components/elements/MobileNav.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { useState } from 'react'
+import { Link } from 'react-router-dom';
+import useDarkMode from 'hooks/dark-mode';
+import Logo from '../../assets/logo.png'
+import lightMode from '../../assets/light-mode.svg'
+import darkMode from '../../assets/dark-mode.svg'
+import closeSymbol from '../../assets/HamX.png'
+import hamMenu from '../../assets/HamReg.png'
+import logoDark from '../../assets/logoDark.png'
+
+
+function MobileNav() {
+  const [menuOpen, setMenuOpen] = useState(false)
+  const [colorTheme, setTheme] = useDarkMode();
+
+
+  return (
+    <>
+      <div className='grid text-right bg-du-gray dark:bg-slate-700 h-16 grid-cols-4 place-content-center place-items-center'>
```

Comment:
comment (blocking):
class naming should be in the order of...

`positioning padding/margins colors other things`

your order right now has text-right next to grid, that needs to be on the other side. Might need to use justify-content or align content